databricks/koalas/generic.py

```diff
@@ -456,118 +456,143 @@ def to_numpy(self):
         """
         return self.to_pandas().values
 
-    def to_csv(self, path_or_buf=None, sep=",", na_rep='', float_format=None,
-               columns=None, header=True, index=True, index_label=None,
-               mode='w', encoding=None, compression='infer', quoting=None,
-               quotechar='"', line_terminator="\n", chunksize=None,
-               tupleize_cols=None, date_format=None, doublequote=True,
-               escapechar=None, decimal='.'):
-        """
+    def to_csv(self, path=None, sep=',', na_rep='', columns=None, header=True,
+               quotechar='"', date_format=None, escapechar=None, num_files=None,
+               **kwargs):
```

Comment:
Hmm .. should I maybe not add this `kwargs` to allow PySpark' option directly for now @ueshin?

Reply:
I think it's okay to have it. Actually I added `options` for `read_sql()/_table()/_query()`.